### DISCUSSION TITLE
Milvus memory consumption

### DISCUSSION THREAD

**Discussion by computerAItest** [milvus-io/milvus#4057] [Q&A and General discussion]
Milvus started by docker, when generating millions of data set, when there is 100 MB of memory remaining, Milvus will stop directly. Is there any requirement for this Milvus? When the memory remaining is relatively small

docker启动的milvus，再生成百万数据底库时候，内存剩下100duo兆时候，milvus直接停止，这个milvus是有什么要求么，在内存剩余比较小情况下，100万数据在生成30万时候内存剩余100M，就退出了， 试了几次都是这样

**Comment by shiyu22** [comment]
Is there have any error logs? Actually, Milvus won't quit despite the small memory, so can you provide logs and describe the details of your operation?
